core/trino-main/src/main/java/io/trino/SystemSessionProperties.java

```diff
@@ -130,7 +129,7 @@
     public static final String COST_ESTIMATION_WORKER_COUNT = "cost_estimation_worker_count";
     public static final String OMIT_DATETIME_TYPE_PRECISION = "omit_datetime_type_precision";
     public static final String USE_LEGACY_WINDOW_FILTER_PUSHDOWN = "use_legacy_window_filter_pushdown";
-    public static final String JOIN_PUSHDOWN_MODE = "join_pushdown";
+    public static final String ENABLE_JOIN_PUSHDOWN = "enable_join_pushdown";
```

Comment:
Wrr. Half of the "enable" flags is "enable_*" while the other half is "*_enabled".
More broadly, some flags  are nouns, some are verbs.........

Keep as is, but at some point we should revisit.

Reply:
dropped